src/components/clusterDetail/ClusterInstallationError.tsx

```diff
@@ -23,9 +24,26 @@ const ClusterInstallationError: React.FC<ClusterInstallationErrorProps> = ({
       variant={AlertVariant.danger}
       title={`Cluster installation failed`}
       actionLinks={
-        <AlertActionLink onClick={() => setResetClusterModalOpen(true)}>
-          Reset Cluster
-        </AlertActionLink>
+        <>
+          <AlertActionLink onClick={() => setResetClusterModalOpen(true)}>
+            Reset Cluster
+          </AlertActionLink>
+          {/* TODO(mlibra): Download cluster installation logs
+          <AlertActionLink
+            onClick={() => {
+            }}
+          >
+            Download Installation Logs
+          </AlertActionLink>
+          */}
+          <AlertActionLink
+            onClick={() => {
+              window.open(getBugzillaLink(cluster.openshiftVersion), '_blank');
+            }}
+          >
+            Create A New Bugzilla
```

Comment:
```suggestion
            Report a bug
```

Reply:
per design but I would agree with this suggestion